### PR TITLE
AsyncIOPool.free_size property implementation

### PR DESCRIPTION
**Pull request by Fogapod**
Also implements `min_size` and `max_size`.

Resolves: #109

### CODE REVIEW

**Comment by Fogapod** [comment]
Documentation preview
image

**Comment by Fogapod** [comment]
@1st1 could you please review this one?

File: docs/api/asyncio_con.rst [hunk 1]
```diff
@@ -530,3 +530,15 @@ Connection Pools
 
         See :py:meth:`AsyncIOConnection.execute()
         <edgedb.AsyncIOConnection.execute>` for details.
+
+    .. py:attribute:: min_size()
```
Comment (1st1) [inline docs/api/asyncio_con.rst:534]:
```suggestion
    .. py:attribute:: min_size
```

File: docs/api/asyncio_con.rst [hunk 2]
```diff
@@ -530,3 +530,15 @@ Connection Pools
...
+
+        Number of connection the pool was initialized with.
+
+    .. py:attribute:: max_size()
```
Comment (1st1) [inline docs/api/asyncio_con.rst:538]:
```suggestion
    .. py:attribute:: max_size
```

File: docs/api/asyncio_con.rst [hunk 3]
```diff
@@ -530,3 +530,15 @@ Connection Pools
...
+
+        Max number of connections in the pool.
+
+    .. py:attribute:: free_size()
```
Comment (1st1) [inline docs/api/asyncio_con.rst:542]:
```suggestion
    .. py:attribute:: free_size
```

**Comment by elprans** [comment]
LGTM

**Comment by 1st1** [comment]
Thanks, @Fogapod!

**Pull request merged**